aspnetcore/blazor/debug.md

```diff
@@ -332,6 +334,7 @@ If you're running into errors, the following tips may help:
 * If your environment uses an HTTP proxy, make sure that `localhost` is included in the proxy bypass settings. This can be done by setting the `NO_PROXY` environment variable in either:
   * The `launchSettings.json` file for the project.
   * At the user or system environment variables level for it to apply to all apps. When using an environment variable, restart Visual Studio for the change to take effect.
+* Ensure that no firewalls or proxies block communication with the debug proxy. If the [Windows Firewall](/windows/security/threat-protection/windows-firewall/windows-firewall-with-advanced-security) is blocking the communication, [create an inbound port rule](/windows/security/threat-protection/windows-firewall/create-an-inbound-port-rule) for any blocked ports.
```

Comment:
This one is gonna be tricky because we intentionally use random ports when communicating with the mono proxy. If possible, I would recommend instructing users to allow "NodeJS" processes to communicate over private networks.